apps/obsidian/src/components/canvas/overlays/RelationPanel.tsx

```diff
@@ -249,6 +249,8 @@ export const RelationsPanel = ({
         src,
         title: file.basename,
         nodeTypeId: nodeTypeId,
+        size: "s",
+        fontFamily: "draw",
```

Comment:
Default needs to be `Sans` throughout.  This was requested when we did this in Roam.  Double check that when a new node is added to the canvas or created for the firs time that is defaults to `sans`.  This was a issue in Roam.

It is also default `m` in Roam, but we can leave that until we consolidate further.

Reply:
My bad, default should be `s` for nodes.  Relations might be `m`
https://github.com/DiscourseGraphs/discourse-graph/pull/432